docs/user-guide/getting-started.ipynb

```diff
@@ -0,0 +1,94 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Getting Started\n",
+    "\n",
+    "## Running Beamlime\n",
+    "\n",
+    "Run the `beamlime` command, and specify which workflow to use.\n",
+    "Usually this would be done from a terminal.\n",
+    "The current setup plots the results of the workflow and saves them to a file, which can we specify using the `--image-path` flag:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!beamlime \\\n",
+    "    --workflow dummy \\\n",
+    "    --image-path reduction-result"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The dummy workflow returns a single result named \"random-counts\", which is added as a suffix to the output file name:\n",
+    "\n",
+    "![Plot of reduction result](reduction-result-random-counts.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Adding workflows\n",
+    "\n",
+    "Python packages can add workflows as plugins to Beamlime.\n",
+    "To do this, create a Python package with a `beamlime.stateless` entry point in the `pyproject.toml` file.\n",
+    "This is how the \"dummy\" workflow is added to Beamlime:\n",
+    "\n",
+    "```python\n",
+    "# In beamlime's pyproject.toml\n",
+    "[project.entry-points.\"beamlime.stateless\"]\n",
+    "dummy = \"beamlime.stateless_workflow:dummy_workflow\"\n",
+    "```\n",
+    "\n",
+    "Above, 'dummy' is the name of the workflow (which can then be passed to `beamlime --workflow`), and 'beamlime.stateless_workflow' is the name of the Python package and module containing the workflow.\n",
+    "As an example, for a Loki workflow plugin provided by `esssans`, the entry point might look as follows:\n",
+    "\n",
+    "```python\n",
+    "# In esssans' pyproject.toml\n",
+    "[project.entry-points.\"beamlime.stateless\"]\n",
+    "ess-loki = \"ess.sans.loki:live_workflow\"\n",
+    "```\n",
+    "\n",
+    "where `live_workflow` must adhere to the `beamlime.StatelessWorkflow` protocol:\n",
```

Comment:
Can we use the link to the api-reference page of ``StatelessWorkflow`` instead of using code snippet here?
Or, print it with ``inspect.getsource``...?

Also, we need to add ``StatelessWorkflow`` in the ``api-reference/index.md``.